### PR TITLE
(maint) Update hands on lab so it works with bolt 0.10.0

### PR DESCRIPTION
**Pull request by adreyer**

### CODE REVIEW

File: 4-running-scripts/README.md [hunk 1]
```diff
@@ -66,14 +66,19 @@ Test-Connection -ComputerName "example.com" -Count 3 -Delay 2 -TTL 255 -BufferSi
...
-$ bolt script run testconnection.ps1 --nodes winrm://<node> --user <user> --password <password>
-node_1:
-
-Source        Destination     IPV4Address      IPV6Address                              Bytes    Time(ms)
-------        -----------     -----------      -----------                              -----    --------
-DESKTOP-G9... example.com     93.184.216.34                                             256      83
-DESKTOP-G9... example.com     93.184.216.34                                             256      85
-DESKTOP-G9... example.com     93.184.216.34                                             256      86
+$ bolt script run testconnection.ps1 --n winrm://vagrant:vagrant@localhost:55985
+Started on localhost...
```
Comment (nicklewis) [inline 4-running-scripts/README.md]: This output should reference `vagrant`, yeah?

File: 2-acquiring-nodes/README.md [hunk 2]
```diff
@@ -126,15 +134,17 @@ docker-compose ps
...
+Note the `Ports` column. We are forwarding a local port to the SSH server running in the container. So you should be able to SSH to `127.0.0.1:32768` (in the example above).
 
 The image sets the username to `root` and the password to `root`. Test the connection out if you have a local SSH client like so, changing the port to one you get from running the `docker-compose ps` command above.
 
 ```
 ssh root@127.0.0.1 -p 32768
 ```
 
-When passing nodes to `bolt` in the next section you will use `--nodes 127.0.0.1:32768,127.0.0.1:32769`, replacing the ports with those you see when you run the `docker-compose ps` command shown above. 
+Make sure you can log into all the nodes before moding on you may have to remove some entries from `~/.ssh/known_hosts`
```
Comment (nicklewis) [inline 2-acquiring-nodes/README.md]: s/moding/moving, and should split into two sentences

File: 5-writing-tasks/README.md [hunk 3]
```diff
@@ -85,36 +84,49 @@ Note:
 
 # Write your first task in Python
 
-The above examples are obviously very simple. Lets implement something slightly more interesting and useful. We'll use Python for this example but remember Puppet Tasks can be implemented in any language which can be run on the target node.
+The above examples are obviously very simple. Lets implement something slightly more interesting and useful. We'll use Python for this example but remember Puppet Tasks can be implemented in any language which can be run on the target node. When using a more more featured language like pyhton tasks can return structured data by printing a single JSON object to stdout. This helps
```
Comment (nicklewis) [inline 5-writing-tasks/README.md]: "more more", s/pyhton/python

File: 6-downloading-and-running-existing-tasks/README.md [hunk 4]
```diff
@@ -24,54 +24,66 @@ It is also useful to have some familiarity with running commands with `bolt` so
...
 
-```bash
-mkdir -p modules/package/tasks
-wget https://raw.githubusercontent.com/puppetlabs/tasks-hands-on-lab/master/6-downloading-and-running-existing-tasks/modules/package/tasks/init.rb
+```
+gem install r10k
 ```
 
-Or the same on Windows with PowerShell:
+Now you can use r10k to install the modues in the Puppetfile `~/.puppetlabs/task-modules/Puppetfile`
```
Comment (nicklewis) [inline 6-downloading-and-running-existing-tasks/README.md]: modules

File: 4-running-scripts/README.md (hunk 1 above)
Comment (adreyer) [reply 4-running-scripts/README.md]: the host is localhost

**Pull request merged**